test/scripts/e2e-cp.sh

```diff
@@ -66,6 +66,13 @@ if [ "$OVN_GATEWAY_MODE" == "shared" ]; then
   SKIPPED_TESTS+="Should ensure load balancer service|LGW"
 fi
 
+if [ "$OVN_GATEWAY_MODE" == "local" ]; then
+  if [ "$SKIPPED_TESTS" != "" ]; then
+    SKIPPED_TESTS+="|"
+  fi
+  SKIPPED_TESTS+="Should ensure connectivity works on an external service when mtu changes in intermediate node"
```

Comment:
why are we skipping it for local gw mode? Shouldn't it work there too?

Reply:
yes, right, it should work for local gw mode as well. 
removed from skip tests now.